### PR TITLE
[PWPA-2142] Add formation temperature to generated alfacase for APB plugin table

### PR DESCRIPTION
**Pull request by gabrielantao**
In this PR I added the data to build the temperature table for the APB plugin. All the measured depths registered in the trajectory of well are used and the temperatures are interpolated for that positions and filled equally for all annuli. The first position of trajectory for the example `score_input_injection_operation.json` was zero but it's inconsistent, it should be 2325 m because of `water_depth` and `air_gap` registered in this file; this was fixed.

PWPA-2142

### CODE REVIEW

**Comment by gabrielantao** [comment]
@noliveiraesss I ask your help here to double check if the names in the output alfacase match.

File: src/alfasim_score/converter/alfacase/convert_plugin_data.py [hunk 1]
```diff
@@ -55,7 +57,33 @@ def get_fluid_id(self, fluid_name: str) -> int:
...
+        measured_depths = Array(
+            [
+                self.get_position_in_well(depth).GetValue()
+                for depth in self.score_input.read_well_trajectory()["md"]
+            ],
+            LENGTH_UNIT,
+        )
+        formation_temperature_data = self.score_input.read_formation_temperatures()
+        trajectory = self.score_input.read_well_trajectory()
+        interpolated_temperatures_y = np.interp(
```
Comment (carlosnewmar) [inline src/alfasim_score/converter/alfacase/convert_plugin_data.py:75]: It seems that `formation_temperature_data["elevations"]` is quota and `trajectory["y"]` is TVD. Thus, it is using incompatible data. Is it correct?

Comment (gabrielantao) [reply src/alfasim_score/converter/alfacase/convert_plugin_data.py:75]: I suppose that's true! Fixed

File: src/alfasim_score/converter/alfacase/score_input_reader.py [hunk 2]
```diff
@@ -50,11 +50,16 @@ def read_general_data(self) -> Dict[str, Any]:
             "air_gap": Scalar(self.input_content["air_gap"], LENGTH_UNIT),
         }
 
-    def read_well_trajectory(self) -> Tuple[Array, Array]:
+    def read_well_trajectory(self) -> Dict[str, Any]:
```
Comment (carlosnewmar) [inline src/alfasim_score/converter/alfacase/score_input_reader.py]: `  -> Dict[str, Array]`

**Review by carlosnewmar — APPROVED** [review]

**Review by noliveiraesss — APPROVED** [review]
Everything seems ok about the alfacases.
Nice work!

**Review by ckiyono — APPROVED** [review]

**Pull request merged**